compiler/rustc_middle/src/ty/instance.rs

```diff
@@ -470,10 +470,33 @@ impl<'tcx> Instance<'tcx> {
     /// This function returns `Some(substs)` in the former case and `None` otherwise -- i.e., if
     /// this function returns `None`, then the MIR body does not require substitution during
     /// codegen.
```

Comment:
```suggestion
    /// This function returns `Some(substs)` in the former case and `None`
    /// otherwise -- i.e., if this function returns `None`, then the MIR body
    /// does not require substitution during codegen.
```
These line ran long in my reader.

Reply:
The formatting already matches column used for wrapping the code. It also matches one of two predominant columns used for wrapping comments in this file. 

Furthermore, the comment is not being modified here, so I see no reason to introduce additional spurious formatting changes.